charter.html

```diff
@@ -328,16 +336,13 @@ <h2 id=specifications>Specifications</h2>
 
 <ul>
 <li>because the Specification has been migrated elsewhere
-<li>because the Specification no longer has the support of
```

Comment:
Instead of removing this option, can we replace it with something like "because the Specification has changed substantially enough that it no longer look like it would lead to two interoperable, implementation experiences." to match the other changes made here?

Reply:
I would prefer not to spend time and energy to have a constant reevaluation of likely implementations. If there is an active editor, we should allow the work to continue. Any concerns around editorial decisions can be made via the decision policy.

Reply:
Since this is a "may" not a "must" for the Chairs ("The Chairs may remove Specifications"), it doesn't require expending energy for constant re-evaluation. It merely allows the option to use this as a rationale. I'd imagine the chairs would remove a Specification for this reason if it's actively brought to their attention that a Specification no longer has enough support.

I'd expect this to most likely happen because some possible implementers change their minds about whether a privacy-related technology is necessary or appropriate. And I'd expect that to be rare.

All that said, I think it is reasonable and useful to let the Chairs use this as one possible rationale for removal.

Reply:
Okay, I've restored this in 12a64a7, though I've reworded it based on @TanviHacks's suggestion and the existing paragraph beginning "The Community Group may adopt a Proposal as a Specification if..."